common/changes/office-ui-fabric-react/groupheader-aria_2019-01-24-00-29.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Exposes props of selection check button in group header",
```

Comment:
My one small suggestion would be to add the affected component to the start of the comment, perhaps something like "GroupHeader: Exposes props of selection check button"

Reply:
I've committed the suggestion - thanks!